lms/envs/production.py

```diff
@@ -1124,9 +1124,15 @@
 # SurveyGizmo settings
 SURVEY_GIZMO_TOKEN = AUTH_TOKENS.get('SURVEY_GIZMO_TOKEN', None)
 SURVEY_GIZMO_TOKEN_SECRET = AUTH_TOKENS.get('SURVEY_GIZMO_TOKEN_SECRET', None)
-	# Django channels settings
+
+# Django channels settings
 CELERY_BROKER_HOSTNAME = ENV_TOKENS.get('CELERY_BROKER_HOSTNAME', None)
 
+# Settings for Google reCaptcha
+CAPTCHA_VERIFY_URL = "https://www.google.com/recaptcha/api/siteverify"
```

Comment:
Can we consider moving all these settings in `common.py`? Because its more meaningful, all settings which are common between all of our servers are placed in common.

Reply:
AUTH_TOKENS was not available in common.py thats why i have added it here.